core/src/main/java/org/apache/carbondata/core/datastore/page/encoding/ColumnPageCodec.java

```diff
@@ -21,15 +21,28 @@
 
 /**
  *  Codec for a column page data, implementation should not keep state across pages,
- *  caller will use the same object to encode multiple pages.
+ *  caller may use the same object to apply multiple pages.
  */
-public interface ColumnCodec {
+public interface ColumnPageCodec {
```

Comment:
We better bring RLE of mdk also to similar interfaces, may we can work on future PR.